bin/stacks/routing-api-stack.ts

```diff
@@ -458,10 +463,6 @@ export class RoutingAPIStack extends cdk.Stack {
     const successRateByRequestSourceAndChainIdAlarm: cdk.aws_cloudwatch.Alarm[] = []
     REQUEST_SOURCES.forEach((requestSource) => {
       SUPPORTED_CHAINS.forEach((chainId) => {
-        if (CHAINS_NOT_MONITORED.includes(chainId)) {
-          return
-        }
-
```

Comment:
should we keep this?

Reply:
I think we should remove it, because per-source per-chain alarm is at sev-3, and we should get alerted on testnets with sev-3. We can adjust accordingly, if we find it too noisy.